.vib/nats/goss/goss.yaml

```diff
@@ -0,0 +1,41 @@
+file:
+  /bitnami/nats/conf/{{ .Vars.natsFilename }}.conf:
+    mode: "0644"
+    filetype: file
+    exists: true
+    owner: root
+command:
+  {{- $running_user := printf "%d" .Vars.containerSecurityContext.runAsUser }}
+  {{- $auth := printf "--user=%s --password=%s" .Vars.auth.user .Vars.auth.password }}
+  {{- $client_port := printf "%d" .Vars.service.ports.client }}
+  {{- $local_endpoint := printf "-s nats://nats:%s" $client_port }}
+  local-key-management:
```

Comment:
This is not really a _local_ key management, you are using the LoadBalancer... I think the naming could lead to some misinterpretation when reading the tests.

Reply:
You are right, I overlooked this. Thanks for the catch!